ui/src/dashboards/components/variablesControlBar/VariableDropdown.tsx

```diff
@@ -0,0 +1,70 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {Dropdown, DropdownMenuColors} from 'src/clockface'
+
+// Styles
+import 'src/dashboards/components/variablesControlBar/VariableDropdown.scss'
+
+interface Props {
+  name: string
+  variableID: string
+  values: string[] //todo
+  initialSelected: string // todo
+  onSelect: (variableID: string, value: string) => void // todo
```

Comment:
`todo`?

Reply:
oh yeah, I just wanted to check with you that the values should be strings

Reply:
oh, yep!